cv.tex

```diff
@@ -44,92 +50,85 @@
 % Education
 %------------------------------------------------
 
-\section{Education} 
-
-\subsection{Cornell University}
-
-\descript{MEng in Computer Science}
-\location{Expected Dec 2014 | Ithaca, NY \\ Cum. GPA: N/A}
+\section{Education}
+\subsection{\href{https://www.ucf.edu/}{UCF}}
+\descript{\href{https://www.ucf.edu/degree/information-technology-bs/}{B.S. in Information Technology}}
+\location{December 2014 | Orlando, Fl}
```

Comment:
(JC):
“Fl” —> “FL”

Reply:
(JC):
In the header section, I mentioned adding extra space before and after the pipe. If you end
up doing that, also apply extra spacing in this section for consistency.

Reply:
ty